service/app/models/publisher.rb

```diff
@@ -0,0 +1,8 @@
+class Publisher < ActiveRecord::Base
+	include ApplicationModel
+
+	has_many  :books
```

Comment:
__[rubocop]__
* Tab detected. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-indentation)
* Put one space between the method name and the first argument.